bemani/backend/jubeat/common.py

```diff
@@ -65,14 +65,56 @@ def handle_lobby_check_request(self, request: Node) -> Node:
         data = Node.void('data')
         root.add_child(data)
 
-        data.add_child(Node.s16('interval', 0))
-        data.add_child(Node.s16('entry_timeout', 0))
-        entrant_nr = Node.u32('entrant_nr', 0)
+        data.add_child(Node.s16('interval', 1))
+        data.add_child(Node.s16('entry_timeout', 15))
+        entrant_nr = Node.u32('entrant_nr', 1)
         entrant_nr.set_attribute('time', '0')
         data.add_child(entrant_nr)
+        waitlist = Node.void('waitlist')
+        waitlist.set_attribute('count', '0')
+        data.add_child(waitlist)
 
         return root
 
+    def handle_lobby_entry_request(self, request: Node) -> Node:
```

Comment:
This looks like you're implementing lobby support (YAY) for jubeat, but I don't see any backend lobby storage. Are we faking it out or is it correct and the state is all client-side?

Reply:
I'm faking it out right now so that I'm not sitting in the online matching screen for 30 seconds. I'm planning on implementing it properly though

Reply:
Ah, I disabled it globally for the older games, and the newer ones I just hacked it out of the state machine, so I never see the wait.